.github/workflows/backend-ci.yml

```
@@ -0,0 +1,94 @@
+name: Backend CI
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - 'backend/**'
+  pull_request:
+    branches:
+      - main
+    paths:
+      - 'backend/**'
```

一応`workflow_dispatch`も追加しておいたほうが良いかなと思いました

```suggestion
      - 'backend/**'
  workflow_dispatch:
```